public/static/less/banner.less

```diff
@@ -0,0 +1,29 @@
+.banner-container {
+    height: 50px;
+    margin: 8px;
+}
+.banner-contents {
```

Comment:
Should this be yellow instead of green?

Reply:
I had yellow at first, but I thought the green looked better. I think you might be leaning toward yellow because it's normally the color associated with warnings, but these aren't necessarily warnings. I'm planning on creating a new feature ticket to allow setting the banner to be (info/warning/important) which will change both the icon and color